cerebrium/endpoints/custom-web-servers.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Custom Web Servers"
+description: "Run ASGI/WSGI python apps on Cerebrium"
+---
+
+While Cerebrium's default runtime works well for most app needs, teams sometimes need more control over their web server implementation. Using ASGI or WSGI servers through Cerebrium's custom runtime feature enables capabilities like custom authentication, dynamic batching, frontend dashboards, public endpoints, and websockets.
+
+## Setting Up Custom Servers
+
+Here's a simple FastAPI server implementation that shows how custom servers work in Cerebrium:
+
+```python
+from fastapi import FastAPI
+
+server = FastAPI()
+
+@server.post("/hello")
+def hello():
+    return {"message": "Hello Cerebrium!"}
+
+@server.get("/health")
+def health():
+    return "Ok"
+```
+
+Configure this server in `cerebrium.toml` by adding a custom runtime section:
+
+```toml
+[cerebrium.runtime.custom]
+port = 5000
+entrypoint = ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "5000"]
```

Comment:
it needs to be main:server based on above implementation

Reply:
Good catch. Changed this instead to be what it is in our examples (And using app)